driver-mapping/src/main/java/com/datastax/driver/mapping/Mapper.java

```diff
@@ -929,6 +934,20 @@ public static Option saveNullFields(boolean enabled) {
             return new SaveNullFields(enabled);
         }
 
+        /**
+         * Creates a new Option object to specify the keyspace to use. This is valid for save, delete and get
+         * operations.
+         * <p/>
+         * If this option is not specified, it defaults to {@code null} (use the keyspace defined in the session or the
+         * {@link com.datastax.driver.mapping.annotations.Table} annotation).
+         *
+         * @param keyspace the name of the keyspace.
+         * @return the option.
+         */
+        public static Option keyspace(String keyspace) {
```

Comment:
If you specify the `keyspace` option, the keyspace annotation will be ignored, yes. Are you suggesting that the Javadoc doesn't make this clear? Please indicate what changes you want me to make.

Please make sure to read the use case in [JAVA-1229](https://datastax-oss.atlassian.net/browse/JAVA-1229).